canonical-web-and-design/snapcraft.io
# Feature: Guided tour for snap listing page

This is a feature branch.
All the code was reviewed in the individual PRs.

### QA
- ./run or https://snapcraft-io-canonical-web-and-design-pr-2220.run.demo.haus/
- sign in, go to listing page of any snap you own, click on tour icon (?) to start the tour
- click around to check if everything works as expected
- tour should highlight elements properly
- highlighted elements should be interactive (accessible with mouse)
- tour should scroll the page if necessary when moving between the steps
